specification/confidentialledger/resource-manager/readme.csharp.md

```diff
@@ -10,6 +10,6 @@ csharp:
   payload-flattening-threshold: 1
   clear-output-folder: true
   client-side-validation: false
-  namespace: Microsoft.ConfidentialLedger
-  output-folder: $(csharp-sdks-folder)/confidentialledger/management/Microsoft.ConfidentialLedger/GeneratedProtocol
+  namespace: Azure.Management.ConfidentialLedger
+  output-folder: $(csharp-sdks-folder)/confidentialledger/Azure.Management.ConfidentialLedger/GeneratedProtocol
```

Comment:
@ArcturusZhang Does this change make sense for csharp ?

Reply:
Updated to Azure.ResourceManager.ConfidentialLedger based on this page.
https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespaces-approved-list

Please let me know if this is more appropriate